config/sidebar-learn.json

```diff
@@ -370,6 +370,11 @@
         "source": "learn/primary_db_sync/meilisync_postgresql.mdx",
         "label": "Syncing databases with PostgreSQL and meilisync",
         "slug": "meilisync_postgresql"
+      },
+      {
+        "source": "learn/cookbooks/laravel-multitenancy.mdx",
```

Comment:
Could we change this and the filename to use underscores instead of hyphens for consistency?